common/persistence/dataStoreInterfaces.go

```diff
@@ -716,7 +716,7 @@ type (
 		CloseTimestamp     time.Time
 		Status             types.WorkflowExecutionCloseStatus
 		HistoryLength      int64
-		RetentionSeconds   time.Duration
+		RetentionPeriod    time.Duration
```

Comment:
minor honestly, but do we have any broad pattern for this kind of thing for "period" vs "interval"?  since it's a bit odd to do it two ways in the same file.

(not that I think either is _wrong_, they kinda are different meanings.  just wondering out loud if consistency would be better.)

Reply:
Yeah I thought about it.

But if we decide to do it, we should change it everywhere. `Period` mostly goes with workflow retention, `Interval` is very common for retry policy fields. We also have lots of `Timeout` fields as well.

This would be big change. I'm not against it, but IMHO this is very low priority item.